src/menu.js

```diff
@@ -458,13 +459,32 @@ exports.get = function(props = {}) {
                 .map(syncer => syncer.id)[0]
 
             if (syncerId == null) {
-              dialog.showMessageBox(
-                i18n.t(
-                  'menu.engines',
-                  'None of the attached engines support analysis.'
-                ),
-                'info'
-              )
+              let defaultEngine = setting
+                .get('engines.list')
+                .filter(engine => engine.analysis)[0]
+
+              if (defaultEngine != null) {
+                if (!analyzeEngineInitializing) {
+                  analyzeEngineInitializing = true
+
+                  sabaki.attachEngines([defaultEngine], ({syncer, error}) => {
+                    if (error != null) {
+                      dialog.showMessageBox(error.message, 'error')
```

Comment:
Unfortunately, `error.message` is untranslatable. Either show a message box with a generic error message, or we ignore errors as before. The user can check the GTP console for errors.

Reply:
I changed to display a generic error message.